lib/l10n/intl_en.arb

```diff
@@ -906,6 +906,26 @@
   "demoSimpleDialogDescription": "A simple dialog offers the user a choice between several options. A simple dialog has an optional title that is displayed above the choices.",
   "@demoSimpleDialogDescription": {
     "description": "Description for the simple dialog component demo."
+  },
+    "demoDividerTitle": "Divider",
+  "@demoDividerTitle": {
+    "description": "Title for the divider component demo."
+  },
+  "demoDividerSubtitle": "A thin horizontal line, with padding on either side",
```

Comment:
Since you should include the vertical divider too, update the subtitle to:

`A divider is a thin line that groups content in lists and layouts.`